akirachix/Upile-Informational
# Adds updates section

**Description**
This PR updates the updates section of our informational website to enhance user engagement and improve information accessibility.
By refreshing this section, we aim to provide timely information, improve user experience, and increase site traffic.
Dependencies: No external dependencies are required for this change.
**What this PR does?**
This PR introduces an updates section on our website, featuring a more user-friendly layout and updated content.
This update aims to streamline the content presentation and ensure that users can easily find relevant updates.
**How are you doing this?**
Redesigned the layout for better readability.
Updated existing content to reflect the latest information.
Added new sections for upcoming events and recent news.
Improved navigation links for easier access to related articles.
**List of updates for features, enhancements, or bugfixes:**
Improved layout design for clarity.
Updated text content with current information.
Added a new filter feature to sort updates by date or category.
Enhanced mobile responsiveness for better viewing on smaller screens.
**Type of change**
- Bug fix (non-breaking change which fixes an issue)
- [New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Optimization
- Improvement
**How can it be tested?**
Navigate to the updates section of the website.
Check that all content is up-to-date and accurate.
View the updates section on various devices (desktop, tablet, mobile) to ensure the responsive design works properly.
**Where is it documented?**
**JIRA Ticket:**
https://wambuinatalygakenye.atlassian.net/browse/UP-117?atlOrigin=eyJpIjoiZTlmMzYxYTY5NGNhNDAxNDhiOGYyMWRiOTliNTUzYjIiLCJwIjoiaiJ9
